src/components/History/Components.js

```diff
@@ -29,49 +29,84 @@ ItemInfo.propTypes = {
 ItemInfo.defaultProps = {
   imgURL: reactJPG,
   title: 'The Great Gatsby',
-  status: 'Requesting',
-  supplement: 'Duration: 1-23 Mar. 2019'
+  status: 'Requesting'
+  // supplement: 'Duration: 1-23 Mar. 2019'
 };
 
 const RequestMessage = props => {
   const [id, setId] = useState(props.message.id);
+  const [showOperation, setShowOperation] = useState(
+    props.message.status === 'ongoing'
+  );
+  const [status, setStatus] = useState(props.message.status);
+  const {createdAt} = props.message.stuff
```

Comment:
Haha I get that we are sort of in a hurry, but could you please rename ```stuff``` to something a little more intelligible?

Reply:
I meant literally stuff. But changed due to "book" is our convention to name a "stuff"